apps/platform/src/app/(main)/(project)/[workspace]/[project]/@overview/page.tsx

```diff
@@ -0,0 +1,170 @@
+            </div>
+          </div>
+
+          <div className="flex flex-1 items-start justify-between rounded-[8px] bg-white/10 p-5 ">
+            <div className="flex flex-col gap-2">
+              <p className="text-[20px] font-[500]">Environments</p>
+              <p className="text-[16px] font-[400] text-[#C9C9C9]">
+                See all the environments of the project!
+              </p>
+            </div>
+            <div className="flex gap-2 bg-[#262626] p-2">
+              <SecretSVG width={16} />
+              {activeProject.environmentCount}
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className="rounded-5 flex flex-1 flex-col gap-2 bg-white/5 p-6 drop-shadow-[0px_1px_2px_rgba(16,24,40,0.06)]">
+        <div className="flex items-center gap-x-2 border-b border-white/20 pb-5">
+          <LockSVG />
+          <p className="text-lg font-[700]">Security Settings</p>
+        </div>
+
+        {!privateKey && (
+          <div className="flex  items-center justify-center rounded-[8px] border border-[#FDE047] p-[10px]">
+            <p className="text-[16px] font-[400] text-[#FDE047]">
+              You haven’t stored any private key with us, so secret values can’t
+              be decrypted.
+            </p>
+          </div>
+        )}
+
+        <div className="flex flex-col gap-3 py-2">
+          <div className="flex flex-1 flex-col gap-3 rounded-[8px] bg-white/10 p-5">
+            <div className="flex flex-col gap-[4px]">
+              <p className="text-[20px] font-[500]">
+                Do you wanna setup private key?
+              </p>
+              <p className="text-[16px] font-[400] text-[#C9C9C9]">
+                Settings up your private key in browser helps you with safely
+                setting up your secret.
+              </p>
+            </div>
+            <LocalKeySetup
+              isStoredOnServer={isKeyStoredOnServer}
+              onOpenSetupDialog={() => setShowAddKeyDialog(true)}
+              privateKey={privateKey}
+            />
+          </div>
+
+          <div className="flex flex-1 flex-col gap-3 rounded-[8px] bg-white/10 p-5">
+            <div className="flex flex-col gap-1">
+              <p className="text-lg font-[500]">
+                Do you wanna regenerate private key?
+              </p>
+              <p className="text-base font-[400] text-[#C9C9C9]">
+                This will help you and your team to shared secrets easily.
+              </p>
+            </div>
+            <RegenerateKey projectSlug={activeProject.slug} />
+          </div>
+
+          <div className="flex flex-1 flex-col gap-3 rounded-[8px] bg-white/10 p-5">
+            <div className="flex flex-col gap-1">
+              <p className="text-lg font-[500]">Store private key with us?</p>
+              <p className="text-base font-[400] text-[#C9C9C9]">
+                This will help you and your team to shared secrets easily.
```

Comment:
Storing the private key with us will enable you and your team to easily view the secrets of this project without needing the hassle of setting the key up in their browsers. Although, this comes with the downside of accidental leaks.